cv32e40p/regress/cv32e40p_full_covg_no_pulp.yaml

```diff
@@ -283,8 +283,8 @@ tests:
     build: uvmt_cv32e40p
     description: Debug test target debug_req at BOOT_SET
     dir: cv32e40p/sim/uvmt
-    cmd: make test COREV=YES TEST=debug_test_boot_set
-    num: 50
+    cmd: make test COREV=YES TEST=debug_test_boot_set SEED=1 VSIM_USER_FLAGS=+fixed_instr_gnt_stall=10
```

Comment:
Fixed OBI timing.

Reply:
this is same question to cv32e40p_debug.yaml above.

Reply:
And same answer.

Reply:
OK